media-sound/jack-smf-utils/jack-smf-utils-1.0-r2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="Utilities for MIDI streams and files using Jack MIDI"
+HOMEPAGE="https://sourceforge.net/projects/jack-smf-utils/"
+SRC_URI="https://sourceforge.net/projects/jack-smf-utils/files/${PN}/${PV}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="lash"
```

Comment:
Readline support is gone now?

Reply:
From a grep/diff the code it appears to be part of the smfsh shell for command editing, that code looks newer in the Gentoo system installed package for smfsh in libsmf. This smfsh binary is the thing that collides. 

The differences do not appear to be jack related from what I see but I could be wrong. This is the reason for removing the readline dependency and in the build system.